docker/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM php:7.3-buster
+FROM php:8-buster
 
 RUN apt-get -y update && apt-get -y install git zip && \
 curl -sS https://getcomposer.org/installer | php && \
 mv composer.phar /usr/local/bin/composer && \
 chmod +x /usr/local/bin/composer && \
-pecl install ast-1.0.4 xdebug && \
+pecl install ast-1.0.10 xdebug && \
```

Comment:
`ast-1.0.4` wouldn't compile for PHP8, `ast-1.0.10` works for all three versions of php (`7.3`, `7.4`, and `8.0`)